### PR TITLE
Add changelog for Release 0.4.20

### PR DESCRIPTION
**Pull request by mnishamk-freiheit**

### CODE REVIEW

**Comment by medo-freiheit** [comment]
Please increase the version too.

**Comment by medo-freiheit** [comment]
please rename it to: Release 0.4.20

**Review by hannesg — APPROVED** [review]

File: services/frontend-service/src/ui/ConfirmationDialog.tsx [hunk 1]
```diff
@@ -156,7 +156,7 @@ export const ConfirmationDialogProvider = (props: ConfirmationDialogProviderProp
             <AlertTitle>Warning: Not deployed to "{undeployedUpstream}" yet!</AlertTitle>
             {[
                 `This version is not yet deployed to "${undeployedUpstream}" environment.`,
-                'Your changes will be overridden by the next release train.',
+                'Your changes may be overridden by the next release train.',
```
Comment (medo-freiheit) [inline services/frontend-service/src/ui/ConfirmationDialog.tsx:159]: I believe this shouldn’t be fixed here, we are trying to keep releases PRs clean and only for releasing

**Review by medo-freiheit — CHANGES_REQUESTED** [review]

File: services/frontend-service/src/ui/ConfirmationDialog.tsx (hunk 1 above)
Comment (mnishamk-freiheit) [reply services/frontend-service/src/ui/ConfirmationDialog.tsx]: removed

**Review by medo-freiheit — APPROVED** [review]

**Pull request merged**